lib/project_types/script/layers/infrastructure/api_clients.rb

```diff
@@ -28,12 +28,12 @@ def initialize(ctx, api_key)
 
           private
 
-          def spin_instance_url
-            workspace = ENV["SPIN_WORKSPACE"]
-            namespace = ENV["SPIN_NAMESPACE"]
-            return if workspace.nil? || namespace.nil?
-
-            "https://script-service.#{workspace}.#{namespace}.us.spin.dev"
+          def script_service_url
+            if ::ShopifyCli::Environment.use_local_partners_instance?
+              LOCAL_INSTANCE_URL
+            elsif ::ShopifyCli::Environment.use_spin_partners_instance?
+              "https://script-service.#{::ShopifyCli::Environment.spin_url}"
+            end
```

Comment:
@mkcny and I talked and this might be something we need to support with the installed version of the CLI. We can simplify the logic as we remove the stubbed and local configurations for scripts and solely rely on spin instances.